src/lib/mappers/abstraction-period.js

```diff
@@ -4,12 +4,15 @@ const AbstractionPeriod = require('../models/abstraction-period');
 
 const dbToModel = dbRow => {
   const abstractionPeriod = new AbstractionPeriod();
-  return abstractionPeriod.fromHash({
-    startDay: dbRow.abstractionPeriodStartDay,
-    startMonth: dbRow.abstractionPeriodStartMonth,
-    endDay: dbRow.abstractionPeriodEndDay,
-    endMonth: dbRow.abstractionPeriodEndMonth
-  });
+  if (dbRow.scheme === 'alcs') {
```

Comment:
Please add an inline comment explaining why this only applies to alcs. E.g. Is it that you can't have an abstraction period as part of SROC?

Reply:
comments added